torch/_refs/__init__.py

```diff
@@ -1447,11 +1447,28 @@ def le(a: TensorLikeType, b: TensorLikeType) -> TensorLikeType:
 )
 def logaddexp(a: TensorLikeType, b: TensorLikeType) -> TensorLikeType:
     # Nb. this implementation does nto distribute the gradients evenly when a == b
-    mask = a >= b
+    mask = torch.real(a) >= torch.real(b)
     max_ = torch.where(mask, a, b)
     min_ = torch.where(mask, b, a)
-    inf_mask = torch.logical_and(torch.isinf(a), a == b)
-    return torch.where(inf_mask, a, max_ + torch.log1p(torch.exp(min_ - max_)))
+    inf_mask = torch.logical_and(
+        torch.isinf(torch.real(a)), torch.real(a) == torch.real(b)
+    )
```

Comment:
This does not follow core, as there it uses `std::isfinite` which is equivalent to `torch.isfinite`, which is not equivalent to `not torch.isinf`.

Reply:
Done in the latest commit. Changed it to `torch.logical_not(torch.isfinite(...))`